doc/changelogs/CHANGELOG_IOJS.md

```diff
@@ -2207,7 +2207,7 @@ _Note: version **1.4.0** was tagged and built but not released. A libuv bug was
 * [[`3b1b4de903`](https://github.com/nodejs/node/commit/3b1b4de903)] - **test**: Timeout#unref() does not return instance (Jan Schär) [joyent/node#9171](https://github.com/joyent/node/pull/9171)
 * [[`becb4e980e`](https://github.com/nodejs/node/commit/becb4e980e)] - **test**: distribute crypto tests into separate files (Brendan Ashworth) [#827](https://github.com/nodejs/node/pull/827)
 * [[`77f35861d0`](https://github.com/nodejs/node/commit/77f35861d0)] - **(SEMVER-MINOR) tls**: more secure defaults (Roman Reiss) [#826](https://github.com/nodejs/node/pull/826)
-* [[`faa687b4be`](https://github.com/nodejs/node/commit/faa687b4be)] - **url**: reslove urls with . and .. (Amir Saboury) [#278](https://github.com/nodejs/node/pull/278)
+* [[`faa687b4be`](https://github.com/nodejs/node/commit/faa687b4be)] - **url**: reslove [sic] urls with . and .. (Amir Saboury) [#278](https://github.com/nodejs/node/pull/278)
```

Comment:
I'm not sure if adding '[sic]' helps in the commit messages in changelogs since the way the commits are laid out it could be seen as being part of the literal commit message.